source/shared/cpp/ObjectModel/CaptionSource.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "pch.h"
+#include "BaseCardElement.h"
+#include "ElementParserRegistration.h"
+#include "Util.h"
+
+namespace AdaptiveCards
+{
+class CaptionSource
+{
+public:
+    CaptionSource();
```

Comment:
Default constructor you've defined in `.cpp` doesn't do anything special, so may be `CaptionSource() = default` will suffice?